lib/parsers/asm-parser.ts

```diff
@@ -333,7 +333,7 @@ export class AsmParser extends AsmRegex implements IAsmParser {
         // Opcode expression here matches LLVM-style opcodes of the form `%blah = opcode`
         this.hasOpcodeRe = /^\s*(%[$.A-Z_a-z][\w$.]*\s*=\s*)?[A-Za-z]/;
         this.instructionRe = /^\s*[A-Za-z]+/;
-        this.identifierFindRe = /([$.@A-Z_a-z"]\w*"?)(?:@\w+)*/g;
+        this.identifierFindRe = /((?!\$\.)[$.@A-Z_a-z"][\w.]*"?)(?:@\w+)*/g;
```

Comment:
32-bit x86 with AT&T syntax uses `$` as a prefix for symbols. I included the negative lookahead for `$.` because there’s a test case that contains `$.LC0`, however there are cases like this https://godbolt.org/z/9vhjhhxPG already didn’t work.